src/commonMain/kotlin/io/ktor/io/ByteArrayBuffer.kt

```diff
@@ -7,15 +7,25 @@ public const val DEFAULT_BUFFER_SIZE: Int = 1024 * 16
 
 public class ByteArrayBuffer(
     array: ByteArray,
-    private val pool: ObjectPool<ByteArray>
-) : Buffer() {
+    readIndex: Int = 0,
```

Comment:
BTW: still no way to create `ByteArrayBuffer` from array, offset and size? So like, wrap part of ByteArray?
Similar how `ByteBuffer.wrap(array, offset, length) works
They should be separate from indices, as indices can be reset to zero, and so wrong data will be used

Reply:
marked as resolved, but I don't see any changes regarding it